server/src/domain/person/person.service.ts

```diff
@@ -98,16 +98,34 @@ export class PersonService {
     return this.repository.create({ ownerId: auth.user.id });
   }
 
-  async reassignFaces(auth: AuthDto, personId: string, dto: AssetFaceUpdateDto): Promise<PersonResponseDto[]> {
-    await this.access.requirePermission(auth, Permission.PERSON_WRITE, personId);
+  async reassignFace(authUser: AuthDto, personId: string, dto: FaceDto): Promise<PersonResponseDto> {
```

Comment:
Wait, I'm confused. Isn't `personId` the parameter from `PUT /face/:id`? Why is it called personId not faceId? The id in the route should correspond to a face.

Reply:
Yes, this is incorrect, I will change that